Clarify NODE_FUNCTION_ALLOW_EXTERNAL usage: wildcard not supported, explicit module names required (#3360)
n8n-io/n8n-docs: docs/hosting/configuration/environment-variables/nodes.md

| `N8N_UNVERIFIED_PACKAGES_ENABLED`        | Boolean          | `true`                        | When `N8N_COMMUNITY_PACKAGES_ENABLED` is true, this variable controls whether to enable the installation and use of unverified community nodes from an NPM registry (true) or not (false).                                        |
| `N8N_VERIFIED_PACKAGES_ENABLED`          | Boolean          | `true`                        | When `N8N_COMMUNITY_PACKAGES_ENABLED` is true, this variable controls whether to show verified community nodes in the nodes panel for installation and use (true) or to hide them (false).                                       |
| `NODE_FUNCTION_ALLOW_BUILTIN`            | String           | -                             | Permit users to import specific built-in modules in the Code node. Use * to allow all. n8n disables importing modules by default.                                                                                                   |

> | `NODE_FUNCTION_ALLOW_EXTERNAL` | String | - | Permit users to import specific external modules (from `n8n/node_modules`) in the Code node. n8n disables importing modules by default. **Note:** You must specify the exact module names, separated by commas. Using `*` (wildcard) will **not** work for this variable. |

Reviewer: ```suggestion
| `NODE_FUNCTION_ALLOW_EXTERNAL` | String | - | Permit users to import specific external modules (from `n8n/node_modules`) in the Code node. n8n disables importing modules by default. Note that you must specify the exact module names, separated by commas. Using `*` (wildcard) doesn't work for this variable. |
```
